app/controllers/users_controller.rb

```diff
@@ -51,7 +51,7 @@ def reorder_call_list
   private
 
   def user_params
-    params.require(:user).permit(:name, :email, :_role)
+    params.require(:user).permit(:name, :email, :role)
```

Comment:
It only impacts the search methods and tests IIRC. Since this codebase is extremely well-tested, it shouldn't be an issue. thanks to @colinxfleming  and his nazi testing.